x-pack/platform/plugins/shared/lens/public/visualizations/xy/xy_suggestions.ts

```diff
@@ -639,3 +654,56 @@ function getScore(
 function getExistingLayer(currentState: XYState | undefined, layerId: string) {
   return currentState && currentState.layers.find((layer) => layer.layerId === layerId);
 }
+
+const XY_LINE_INCOMPATIBLE_AGGS = ['count', 'count_distinct', 'cardinality', 'bucket'];
```

Comment:
Should `count`, `count_distinct'`, `cardinality` be disregarded? I saw this in another POC and didn't spend too much time trying to understand why these aggs are not valid for line charts